migrations/2024-08-27-184211_create-exercises-table/up.sql

```diff
@@ -0,0 +1,13 @@
+-- Your SQL goes here
+CREATE TABLE exercises (
+    id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    title VARCHAR(255) NOT NULL,
+    description TEXT NOT NULL,
+    difficulty VARCHAR(255) CHECK (difficulty IN ('easy', 'medium', 'hard')) NOT NULL,
+    test_runner VARCHAR(255) NOT NULL,
+    challenge_id UUID NOT NULL REFERENCES challenges(id),
+    created_at TIMESTAMP NOT NULL DEFAULT now(),
+    updated_at TIMESTAMP NOT NULL DEFAULT now()
```

Comment:
This will be set to datetime at time of row creation and will need to be updated manually on every update after that (unless we add a database trigger).  Just something to note.